client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -144,11 +147,17 @@ class EmailProvidersComparison extends React.Component {
 	};
 
 	goToEmailForwarding = () => {
-		const { currentRoute, selectedDomainName, selectedSiteSlug } = this.props;
+		const { currentRoute, selectedDomainName, selectedSite } = this.props;
 
 		recordTracksEvent( 'calypso_email_providers_add_click', { provider: 'email-forwarding' } );
 
-		page( emailManagementForwarding( selectedSiteSlug, selectedDomainName, currentRoute ) );
+		page( emailManagementForwarding( selectedSite.slug, selectedDomainName, currentRoute ) );
+	};
+
+	isUpgrading = () => {
+		const { domain } = this.props;
+
+		return hasEmailForwards( domain );
```

Comment:
Would a one-liner suffice here?

```suggestion
	isUpgrading = () => hasEmailForwards( this.props.domain );
```